helm/charts/hydra/templates/job-migrations-custom.yaml

```diff
@@ -0,0 +1,128 @@
+{{- range $index, $job := $.Values.hydra.customMigrations.jobs }}
+{{- if $job.enabled -}}
+{{- $nodeSelector := ternary $job.nodeSelector $.Values.job.nodeSelector (not (empty $job.nodeSelector)) -}}
+{{- $migrationExtraEnv := ternary $job.extraEnv $.Values.job.extraEnv (not (empty $job.extraEnv)) -}}
+{{- $resources := ternary $job.resources $.Values.job.resources (not (empty $job.resources)) -}}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "hydra.fullname" $ }}-{{ $job.name | default (printf "automigrate-%d" $index) }}
+  {{- if $.Release.Namespace }}
+  namespace: {{ $.Release.Namespace }}
+  {{- end }}
+  labels:
+    {{- include "hydra.labels" $ | nindent 4 }}
+    {{- with $.Values.job.labels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- with $.Values.job.annotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  template:
+    metadata:
+      annotations:
+        {{- with $.Values.job.annotations }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        {{- with $.Values.job.podMetadata.annotations }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+      labels:
+        app.kubernetes.io/name: {{ include "hydra.fullname" $ }}-automigrate
```

Comment:
label should be also based on custom job name to avoid conflicts with main migration job